src/high-level/tests/KinDynComputationsUnitTest.cpp

```diff
@@ -681,6 +833,14 @@ int main()
     testModelConsistencyAllRepresentations("iCubGenova02.urdf");
     testModelConsistencyAllRepresentations("icalibrate.urdf");
 
+    testModelConsistencyAllRepresentations("oneLink.urdf");
+    testModelConsistencyAllRepresentations("twoLinks.urdf");
+    testSubModelConsistencyAllRepresentations("threeLinks.urdf");
+    testModelConsistencyAllRepresentations("bigman.urdf");
+    testModelConsistencyAllRepresentations("icub_skin_frames.urdf");
+    testModelConsistencyAllRepresentations("iCubGenova02.urdf");
+    testModelConsistencyAllRepresentations("icalibrate.urdf");
```

Comment:
It is intended to only have `testSubModelConsistencyAllRepresentations` for one of the models? If so, probably we can remove the `testModelConsistencyAllRepresentations` for the other models.

Reply:
Maybe one model is enough, I can remove the others.